front_file_hosting/src/App.vue

```diff
@@ -1,30 +1,88 @@
 <template>
   <div id="nav">
     <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
+    <router-link to="/register">Sign up</router-link> |
+    <router-link to="/login">log in</router-link> |
+    <router-link to="/user">User</router-link> |
+    <router-link to="/user/change_password">Change Password</router-link> |
+    <router-link to="/files">Files</router-link> |
+    <router-link to="/upload">Upload</router-link> |
+    <router-link to="/file_detail">File</router-link> |
```

Comment:
minor
I don't know how the value in `to` prop is used, but if they are parts of URL, would be better to change `_` to `-` ([an article about hyphens and underscores](https://www.seomechanic.com/seo-101-hyphens-underscores-_-urls/))